File: CODIGO/AdoptaYA-WebAPI/src/models/bitacora.js
const { DataTypes } = require('sequelize');

module.exports = (sequelize) => {
  const Bitacora = sequelize.define('bitacora', {
    id:         { type: DataTypes.INTEGER.UNSIGNED, primaryKey: true, autoIncrement: true, allowNull: false },
    tabla:      { type: DataTypes.STRING(50), allowNull: true },
    accion:     { type: DataTypes.STRING(10), allowNull: true },
    fecha:      { type: DataTypes.DATE, allowNull: true, defaultValue: DataTypes.NOW },
    datos:      { type: DataTypes.TEXT, allowNull: true },
    id_usuario: { type: DataTypes.INTEGER.UNSIGNED, allowNull: false }
  }, {
    tableName: 'bitacora',
    timestamps: false
  });

  Bitacora.associate = (models) => {
    Bitacora.belongsTo(models.usuario, { foreignKey: 'id_usuario' });
  };

  return Bitacora;
};

File: CODIGO/AdoptaYA/Functionalities/Log/Model/Bitacora.cs
using System.Text.Json.Serialization;

namespace AdoptaYA.Functionalities.Log.Model;
public class Bitacora
{
    [JsonPropertyName("id")]
    public int Id { get; set; }

    [JsonPropertyName("tabla")]
    public string? Table { get; set; }

    [JsonPropertyName("accion")]
    public string? Action { get; set; }

    [JsonPropertyName("fecha")]
    public DateTime Date { get; set; }

    [JsonPropertyName("datos")]
    public string? Data { get; set; }

    [JsonPropertyName("id_usuario")]
    public int UserId { get; set; }

    [JsonPropertyName("usuario")]
    public BitacoraUsuario? User { get; set; }
}